chutney/server/src/main/java/com/chutneytesting/security/api/SsoOpenIdConnectController.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * SPDX-FileCopyrightText: 2017-2024 Enedis
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ */
+
+package com.chutneytesting.security.api;
+
+import com.chutneytesting.security.infra.sso.SsoOpenIdConnectConfig;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping(SsoOpenIdConnectController.BASE_URL)
+@CrossOrigin(origins = "*")
+public class SsoOpenIdConnectController {
+
+    public static final String BASE_URL = "/api/v1/sso";
```

Comment:
may be use `SsoOpenIdConnectController.BASE_URL`

Reply:
I don't understand, this is SsoOpenIdConnectController.BASE_URL